File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/invs/INV0101U01GridMasterInfo.java
package nta.med.data.model.ihis.invs;

public class INV0101U01GridMasterInfo {
	 private String codeType;
	 private String codeTypeName;
	 private String adminGubun;

	public INV0101U01GridMasterInfo(String codeType, String codeTypeName, String adminGubun) {
		super();
		this.codeType = codeType;
		this.codeTypeName = codeTypeName;
		this.adminGubun = adminGubun;
	}

	public String getCodeType() {
		return codeType;
	}

	public void setCodeType(String codeType) {
		this.codeType = codeType;
	}

	public String getCodeTypeName() {
		return codeTypeName;
	}

	public void setCodeTypeName(String codeTypeName) {
		this.codeTypeName = codeTypeName;
	}

	public String getAdminGubun() {
		return adminGubun;
	}

	public void setAdminGubun(String adminGubun) {
		this.adminGubun = adminGubun;
	}
	 
}

File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Invs/INV0101U01GridMasterInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Invs
{
    public class INV0101U01GridMasterInfo
    {
        private String _codeType;
        private String _codeTypeName;
        private String _adminGubun;
        private String _rowState;

        public String CodeType
        {
            get { return this._codeType; }
            set { this._codeType = value; }
        }

        public String CodeTypeName
        {
            get { return this._codeTypeName; }
            set { this._codeTypeName = value; }
        }

        public String AdminGubun
        {
            get { return this._adminGubun; }
            set { this._adminGubun = value; }
        }

        public String RowState
        {
            get { return this._rowState; }
            set { this._rowState = value; }
        }

        public INV0101U01GridMasterInfo() { }

        public INV0101U01GridMasterInfo(String codeType, String codeTypeName, String adminGubun, String rowState)
        {
            this._codeType = codeType;
            this._codeTypeName = codeTypeName;
            this._adminGubun = adminGubun;
            this._rowState = rowState;
        }

    }
}